tests/libraries/token/test_stake_token.py

```diff
@@ -0,0 +1,325 @@
+from ethereum.tools import tester
+from datetime import timedelta
+from utils import longToHexString, stringToBytes
+from pytest import fixture, raises
+from ethereum.tools.tester import ABIContract, TransactionFailed
+from reporting_utils import proceedToDesignatedReporting, proceedToRound1Reporting, proceedToRound2Reporting, proceedToForking, finalizeForkingMarket, initializeReportingFixture
+
+def test_stake_token_creation_binary(tokenFixture, binaryMarket):
+    bin_market = binaryMarket
+    numTicks = 10 ** 18
+    stakeToken_bin = tokenFixture.upload('../source/contracts/reporting/StakeToken.sol', 'stakeToken')
+    # stake token needs same payout numerators as market
+    with raises(TransactionFailed, message="payout numerators can not be more than market"):
```

Comment:
Style: Recommend either including a comment above these raise clauses or including a message.  Having both the comment and the message ends up being a bit redundant.